autojenkins/jobs.py

```diff
@@ -351,3 +352,36 @@ def wait_for_build(self, jobname, poll_interval=3):
             sys.stdout.write('.')
             sys.stdout.flush()
         print('')
+        
+    def createJobIntoView(self, jobname, viewname, config_file, **context):
```

Comment:
I'd rather have the view name as an optional argument in the existing `create` method (`viewname=None` as default).
It makes the API simpler IMO. It also means less duplication of code.

What do you think?